src/functions-and-arrays.js

```diff
@@ -1,19 +1,61 @@
 // Iteration #1: Find the maximum
-function maxOfTwoNumbers() {}
+function maxOfTwoNumbers(name1,name2) {
+  if (name1<name2){
+    return name2;
+  }
+  else if(name1>name2){
+    return name1;
+  }
+  else if(name1=name2){
+    return name1;
+  }
+}
 
+let name1=2
+let name2=1
+  
 
 
 // Iteration #2: Find longest word
 const words = ['mystery', 'brother', 'aviator', 'crocodile', 'pearl', 'orchard', 'crackpot'];
 
-function findLongestWord() {}
+function findLongestWord(arrayOfWords) {
+  let word1=""
+  if (!arrayOfWords.length){
+    return null;
+  }
+  else if(arrayOfWords.length===1){
+    return arrayOfWords[0]
+  }
+  for (let i=0;i<arrayOfWords.length;i++){
+    if (word1.length<arrayOfWords[i].length){
+      word1=arrayOfWords[i]
+    } 
+} return word1;
+}
 
 
 
 // Iteration #3: Calculate the sum
 const numbers = [6, 12, 1, 18, 13, 16, 2, 1, 8, 10];
 
-function sumNumbers() {}
+function sumNumbers(array) {
+  let sum =0
+  if (array.length==="")
```

Comment:
The property `.length` will return you a number, so it is better to give it the value of `0` instead of an empty string:

`if ( array.length === 0 )`